app/models/manageiq/providers/ibm_cloud/vpc/cloud_manager/logging_mixin.rb

```diff
@@ -0,0 +1,85 @@
+# frozen_string_literal: true
+
+# Encapsulate logging methods into a single mixin.
+module ManageIQ::Providers::IbmCloud::VPC::CloudManager::LoggingMixin
```

Comment:
Hey @jaredm-ibm this looks like a change which should be extracted and discussed in its own PR

I think it isn't necessary due to how broadcast logging is handled in containers and appliances but I don't want to hold up the provisioning PR while we discuss that.

If there are other issues with `_log` that you're working around it'd be great to try to solve those generally for everyone.

Reply:
When I was trying to debug that issue with the VM state failing, none of the log messages to ibm_cloud_log were showing up. So I did this.

Reply:
I commented out the 
```
     private_class_method def self.create_container_logger
-      return unless ENV["CONTAINER"]
-
       require "manageiq/loggers/container"
       ManageIQ::Loggers::Container.new
     end
```

as a test and:
```
> $ibm_cloud_log.info("hello world")
{"@timestamp":"2021-04-30T11:35:21.416641 ","pid":170242,"tid":"1773c","level":"info","message":"hello world
```

is printed to stdout.

Reply:
At the moment the error flow uses this implementation. I don't want to rethink it right now. If a better solution comes along then this can be ripped out and rethought.